assets/js/app.js

```diff
@@ -51,6 +51,15 @@ function renderBadges(badges) {
     badges.badges.forEach((badge) => {
         badgesContainer.appendChild(makeElement(renderBadgeContainer(badge)));
     });
+
+    let copyAllBadges = document.querySelector(".copy");
+    let badgesAsMarkdown = "";
+
+    removeChildren(copyAllBadges);
```

Comment:
how about moving it after populating `badgesAsMarkdown `?

Reply:
I will remove badgesAsMarkdown variable and the extra loop there is a better/clean way.

For consistency reason, the new code should follow same flow as the previous code blocks. IMO

- select the element
- remove the element
- populate the dom with a new element

WDYT?

Reply:
Amazing!!! 🚀
Thanks a lot!!!